deeppavlov/skills/aiml_skill/aiml_skill.py

```diff
@@ -88,7 +88,7 @@ def _generate_user_id(self) -> str:
         """
         return uuid.uuid1().hex
 
-    def __call__(self, utterances_batch: List, history_batch: List, states_batch: List) -> Tuple[List, ...]:
+    def __call__(self, utterances_batch: List, history_batch: List=(), states_batch: List=()) -> Tuple[List, ...]:
```

Comment:
`()` is not a `List`
```suggestion
    def __call__(self, utterances_batch: List, history_batch: Optional[List]=None, states_batch: Optional[List]=None) -> Tuple[List, ...]:
```

Reply:
Fixed

Reply:
Fixed